Promact.OAuth.Client/src/Promact.OAuth.Client/Util/StringConstant/StringConstant.cs

```diff
@@ -208,5 +208,249 @@ public string AccessTokenNotMatchedException
             }
         }
         #endregion
+
+        #region Test Cases
+        /// <summary>
+        /// Ramdon access token for test cases
+        /// </summary>
+        public string AccessTokenForTest
+        {
+            get
+            {
+                return "a0a21d71-9f21-41e7-9904-aabe57599554";
+            }
+        }
+
+        /// <summary>
+        /// Random user Id for test cases
+        /// </summary>
+        public string UserIdForTest
+        {
+            get
+            {
+                return "7092802a-206e-4ce4-8192-f086a1ec1fb6";
+            }
+        }
+
+        /// <summary>
+        /// Random slack user Id for test cases
+        /// </summary>
+        public string SlackUserIdForTest
+        {
+            get
+            {
+                return "U0HJKJ4";
+            }
+        }
+
+        /// <summary>
+        /// Random slack group name for test
+        /// </summary>
+        public string SlackGroupNameForTest
+        {
+            get
+            {
+                return "OAuth-Test";
+            }
+        }
+
+        /// <summary>
+        /// Random project Id for test cases
+        /// </summary>
+        public string ProjectIdForTest
+        {
+            get
+            {
+                return "1";
+            }
+        }
+
+        /// <summary>
+        /// GetListOfPromactManagementDetailsAsync Random Value For Test cases
+        /// </summary>
+        public string GetListOfPromactManagementDetailsAsyncRandomValueForTest
+        {
+            get
+            {
+                return "[{\"firstName\":\"Siddhartha\",\"lastName\":null,\"isActive\":false,\"numberOfCasualLeave\":0.0,\"numberOfSickLeave\":0.0,\"joiningDate\":\"0001-01-01T00:00:00\",\"slackUserId\":\"U0HJ49KJ4\",\"slackUserName\":null,\"projects\":null,\"createdBy\":null,\"createdDateTime\":\"0001-01-01T00:00:00\",\"updatedBy\":null,\"updatedDateTime\":\"0001-01-01T00:00:00\",\"id\":\"c2e7cb63-63fb-4c00-a490-a03a8675f66e\",\"userName\":null,\"normalizedUserName\":null,\"email\":\"siddhartha@promactinfo.com\",\"normalizedEmail\":null,\"emailConfirmed\":false,\"passwordHash\":null,\"securityStamp\":null,\"concurrencyStamp\":\"a8aa0e04-dfa3-4589-98e8-b191517e607e\",\"phoneNumber\":null,\"phoneNumberConfirmed\":false,\"twoFactorEnabled\":false,\"lockoutEnd\":null,\"lockoutEnabled\":false,\"accessFailedCount\":0,\"roles\":[],\"claims\":[],\"logins\":[]}]";
```

Comment:
I have used application class for serialization later on method. This is used to return value from mock of http client wrapper.